GetQueueSpidMessages/function.json

```
@@ -0,0 +1,13 @@
+{
+  "bindings": [
+    {
+      "queueName": "spid-msg-items",
+      "connection": "AzureWebJobsStorage",
```

you should use another storage container instead of the default
```suggestion
      "connection": "QueueStorageConnection",
```